akka-projection-jdbc/src/main/scala/akka/projection/jdbc/javadsl/JdbcHandler.scala

```diff
@@ -23,8 +26,13 @@ object JdbcHandler {
     override def process(session: S, envelope: Envelope): Unit = handler(session, envelope)
   }
 
+  @Deprecated
+  @deprecated("Use the similar method with Java functional interface as a parameter", "1.2.0")
```

Comment:
1.2.0 has already been released so this should be 1.2.1

Reply:
🆗